src/test/tests.ts

```diff
@@ -0,0 +1,64 @@
+import { assert } from 'chai'
+import { LoadingOptions, RecordSchema, Simple_schema, ValidationException } from '../util/internal'
+import URL from 'url'
+import path from 'path'
+
+describe('Test RecordSchema', () => {
```

Comment:
Do you have a way to measure code coverage from your tests?

Reply:
> It would probably be useful to have some github action/bot for this.

Can do! We use codecov.io for other CWL projects

Can you 
1. add something like https://github.com/codecov/example-typescript#nyc
2. make a `.github/workflows/ci-tests.yml` file with contents from https://github.com/codecov/codecov-action#usage (and some build steps); the `token` line should be removed, since this is a public repo

For 2, see https://github.com/common-workflow-language/schema_salad/blob/main/.github/workflows/ci-tests.yml for a complicated example (you don't need a `strategy` `matrix` , nor `env`)